tests/test_bundle.py

```diff
@@ -202,3 +208,44 @@ async def test_cloud_events_player_example(
     assert post_req.status_code == 202
     get_req = requests.get(f"{url}/messages", allow_redirects=False, verify=False)
     assert get_req.status_code == 200
+
+
+@pytest_asyncio.fixture
+async def restore_serving_custom_image_settings(ops_test: OpsTest):
+    """Saves the current custom_image setting for serving, restoring it after test completes."""
```

Comment:
Are we restoring the default settings in the case we add another test case that requires a working `knative-serving`?

Reply:
Exactly